jenkins-x-lint.yml

```diff
@@ -14,12 +14,16 @@ pipelineConfig:
             sh: "./add-pr-build-comment"
         - name: lint-checks
           parallel:
+          - name: pip-install-
+               command: python3
+               args:
+               - -m pip install -r python/requirements-dev.txt
           - name: python-lint-task
             steps:
```

Comment:
However, this one could be changed to `parallel` (not that it's gonna make a massive difference though).